style50/__main__.py

```diff
@@ -54,7 +54,7 @@ def main():
                         help="specify output mode. Valid modes are character (default), split, unified, score, and json")
     parser.add_argument("-v", "--verbose", action="store_true",
                         help="print full tracebacks of errors")
-    parser.add_argument("--version", action="version", version="%(prog)s {}".format(__version__))
+    parser.add_argument("-V", "--version", action="version", version="%(prog)s {}".format(__version__))
```

Comment:
It does output the program name, yes. What is the connection to `astyle` here?

Also, `__version__` is the standard way to specify version ([PEP8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names))